src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -92,19 +110,69 @@ public void deployBuddyBalance() {
   }
 
   public void retrieveBuddy() { // Used to pick up the buddy robot while the lift is already underneath it
-    rightPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
+    rightPIDController.setSetpoint(kRetrievedAngle);
+    leftPIDController.setSetpoint(kRetrievedAngle);
+    //buddyBalancePosEntry.setString("Raised");
```

Comment:
Why is this commented out? Let's decide if we want to use this or not.  If it's for a future feature, mark it with a TODO so we don't lose it

Reply:
We marked it with a todo, we need the comp-shuffleboard pull request to go through in order to use it.

Reply:
Uncomment this code and use the matchTab suggestion